client/.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
+    "parser":  "@typescript-eslint/parser",
+    "extends": "airbnb",
+    "plugins": [
+        "react",
+        "jsx-a11y",
+        "import",
+        "@typescript-eslint"
+    ],
+    "env": {
+        "browser": true
+    },
+    "settings": {
+        "import/resolver": {
+            "node": {
+                "paths": ["src"],
+                "extensions": [".js", ".jsx", ".ts", ".tsx"]
+            }
+        }
+    },
+    "rules": {
+        "indent": [2, "tab"],
+        "no-tabs": 0,
+        "react/jsx-indent": [2, "tab"],
+        "react/jsx-indent-props": [2, "tab"],
+        "react/jsx-filename-extension": [0, { "extensions": [".js", ".jsx"] }]
```

Comment:
Should probably include .tsx here if we are going to be using Typescript since .tsx is implicitly for typescript files including jsx.

Reply:
Removed typescript.